proto/status.proto

```diff
@@ -18,11 +19,13 @@ message Status {
   oneof has_bbox {
     string bbox = 5;
   }
-  oneof has_version {
-    string version = 6;
+  string version = 6;
+  uint32 tileset_last_modified = 7;
+  repeated string available_actions = 8;
```

Comment:
these didn't need a `oneof` actually

Reply:
neither do the new ones

Reply:
Ah not @kevinkreiser they're needed, just as the other ones which are only available via `"verbose": true`: in the serializer we're using their `has_` method to check if you should set it in the output JSON or leave it out. I guess that wouldn't necessarily need to be handled with `oneof`s, one could also look at the `verbose()` in the serializer, but that way it'd be tightly coupled to `verbose()` while now we're more flexible and could disallow including them in the JSON based on other logic.